File: java/blackjack/Deck.java
package blackjack;

import java.util.ArrayList;
import java.util.Collections;

/**
 * Created by 1mike12 on 12/16/2016.
 */
public class Deck {

    ArrayList<Card> cards = new ArrayList();

    Deck() {

        for (int i = 1; i < 14; i++) {
            Card newCard = new Card("H", i);
            cards.add(newCard);
        }

        for (int i = 1; i < 14; i++) {
            Card newCard = new Card("S", i);
            cards.add(newCard);
        }

        for (int i = 1; i < 14; i++) {
            Card newCard = new Card("D", i);
            cards.add(newCard);
        }

        for (int i = 1; i < 14; i++) {
            Card newCard = new Card("C", i);
            cards.add(newCard);
        }

        Collections.shuffle(cards);
    }

    Card drawCard() {
        Card theRemovedCard = cards.remove(0);
        return theRemovedCard;
    }
}

File: python2/blackjack/Deck.py
from Card import Card
from random import shuffle

class Deck(object):

    def __init__(self):
        # list of 52 cards, with 1 of each rank and suite
        self.cards = []

        for i in xrange(1,14):
            new_card = Card("S", i)
            self.cards.append(new_card)

        for i in xrange(1,14):
            new_card = Card("C", i)
            self.cards.append(new_card)

        for i in xrange(1,14):
            new_card = Card("D", i)
            self.cards.append(new_card)

        for i in xrange(1,14):
            new_card = Card("H", i)
            self.cards.append(new_card)

        shuffle(self.cards)

    def draw_card(self):
        card = self.cards[0]
        del self.cards[0]
        return card
